src/stats/common.ts

```diff
@@ -224,13 +219,13 @@ export function iterateFramesInOrder(
   const sortedFrames = getSortedFrames(game);
 
   // Iterates through both of the player/opponent pairs
-  _.forEach(opponentIndices, (indices) => {
+  opponentIndices.forEach(indices => {
     initialize(indices);
 
     // Iterates through all of the frames for the current player and opponent
-    _.forEach(sortedFrames, (frame) => {
-      const playerPostFrame = _.get(frame, ['players', indices.playerIndex, 'post']);
-      const oppPostFrame = _.get(frame, ['players', indices.opponentIndex, 'post']);
+    sortedFrames.forEach(frame => {
+      const playerPostFrame = frame.players[indices.playerIndex].post;
+      const oppPostFrame = frame.players[indices.opponentIndex].post;
```

Comment:
I think there was a reason this was previously written with get. Potentially in real-time, you might have received frame data from one player but not the other. I think this might cause an error in the new code because `players[indices.opponentIndex]` would return undefined.

Reply:
I've reverted back to the original code.